### PR TITLE
Russian localization

### PR DESCRIPTION
**Pull request by e-shl**

### CODE REVIEW

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -175,6 +175,7 @@
     <string name="pref_enable_acra">Отсылать отчеты о падениях</string>
     <string name="pref_page_transitions">Анимированные переходы страниц</string>
     <string name="pref_fullscreen">Полноэкранный режим</string>
+    <string name="pref_dual_page_split">Двойное разделение страниц (ALPHA)</string>
```
Comment (jobobby04) [inline app/src/main/res/values-ru/strings.xml:178]: Can you remove the changes in this file? They are from Tachi main and I am planning on keeping these the same. If you want to translate them I suggest doing it through Weblate https://tachiyomi.org/help/contribution/#translation

**Review by jobobby04 — CHANGES_REQUESTED** [review]

**Comment by e-shl** [comment]
what?

**Comment by jobobby04** [comment]
> 
> 
> what?

Move the SY specific translations(theme_dark_midnightdusk and theme_dark_red) from strings.xml to the strings_sy.xml. And remove the rest so that they are managed by Main tachiyomi. I am leaving strings.xml for main Tachi.

**Comment by e-shl** [comment]
Made

**Review by jobobby04 — APPROVED** [review]

**Comment by jobobby04** [comment]
Thanks! I'll merge it before I release the next preview build.

**Pull request merged**